<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialerWidget</class>
 <widget class="QFrame" name="DialerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>179</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="lneNumber">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Enter number</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="hspButtonsLeftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gltButtons">
       <property name="horizontalSpacing">
        <number>4</number>
       </property>
       <property name="verticalSpacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="tlbNumber_1">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">1     </string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="tlbNumber_2">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">2  ABC</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="tlbNumber_3">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">3  DEF</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="tlbNumber_4">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">4  GHI</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="tlbNumber_5">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">5  JKL</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="tlbNumber_6">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">6  MNO</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="tlbNumber_7">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">7 PQRS</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="tlbNumber_8">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">8  TUV</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="tlbNumber_9">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">9 WXYZ</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="tlbNumber_10">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">*     </string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="tlbNumber_11">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">0     </string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="tlbNumber_12">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string notr="true">#     </string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="hspButtonsRightSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vspVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lneNumber</tabstop>
  <tabstop>tlbNumber_1</tabstop>
  <tabstop>tlbNumber_2</tabstop>
  <tabstop>tlbNumber_3</tabstop>
  <tabstop>tlbNumber_4</tabstop>
  <tabstop>tlbNumber_5</tabstop>
  <tabstop>tlbNumber_6</tabstop>
  <tabstop>tlbNumber_7</tabstop>
  <tabstop>tlbNumber_8</tabstop>
  <tabstop>tlbNumber_9</tabstop>
  <tabstop>tlbNumber_10</tabstop>
  <tabstop>tlbNumber_11</tabstop>
  <tabstop>tlbNumber_12</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
